_shared_content/develop/guides/playbook_modules/quick_start.md

```diff
@@ -0,0 +1,234 @@
+# Quick start
+
+In this section we will see how to create a simple module from scratch.
+
+This module will conain one action and one trigger:
+
+* The action will allow to send an HTTP request and return its response.
+* The trigger will watch for ??
```

Comment:
Fixme

Reply:
I'm looking for an idea of q simple trigger we could add to the quick start :sweat_smile: